src/main.cpp

```diff
@@ -5038,14 +5052,19 @@ CBlockTemplate* CreateNewBlock(CReserveKey& reservekey)
                     mapDependers[txin.prevout.hash].push_back(porphan);
                     porphan->setDependsOn.insert(txin.prevout.hash);
                     const CTransaction &txPrevIn = mempool.mapTx[txin.prevout.hash];
-                    nTotalIn += GetPresentValue(txPrevIn, txPrevIn.vout[txin.prevout.n], tx.nRefHeight);
+                    mpq nValueIn = GetPresentValue(txPrevIn, txPrevIn.vout[txin.prevout.n], tx.nRefHeight);
+                    if (fTruncateInputs)
```

Comment:
It's a throwaway code line (since 0.13 will handle it differently), so I thought it better to be fully explicit in what it does with a named variable, even if it is constant in some places and optimized away by the compiler.